src/Components/AllIssues.js

```diff
@@ -0,0 +1,325 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import FormDialog from "./Dialog";
+import AddNewProject from "./AddNewProject";
+import AuthenticationService from "../Services/AuthenticationService";
+import Pagination from "./Pagination";
+import { FaSortDown, FaSortUp } from "react-icons/fa";
+import { Button, ButtonGroup } from "@mui/material";
+import ParticularIssueDashboard from "./ParticularIssueDashboard";
+import MyIssues from "./Myissues";
+import { GridLoader ,HashLoader,PacmanLoader } from "react-spinners";
+import Navbar from "./Navbar";
+import CreateIssueForm from "./CreateIssueForm";
+
+
+export const AllIssues = () => 
+{
+  const maxWidth = "lg";
+  const [filterStatus, setFilterStatus] = useState("All");
+  const [allList, setAllList] = useState([]);
+  const [myIssues, setMyIssues] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [flag , setFlag] = useState('false');
+  const ProjectID = Number(localStorage.getItem("ProjectID"));
+  //console.log("userID",userID);
+  var payload = {project_id: ProjectID};
+  console.log(payload);
+  useEffect(()=>{
+    AuthenticationService.allIssues(payload).then((response)=>{
+      console.log(response.data);
+    }).catch((error)=>{
+      console.log(error);
+    })
+  },[])
+  
```

Comment:
You can remove this commented code if not required in future

Reply:
I have remove the unwanted Part